src/main/java/edu/kis/powp/jobs2d/drivers/command/OperateToCommand.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.jobs2d.drivers.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class OperateToCommand implements DriverCommand {
+    private final int x, y;
+    private final Job2dDriver driver;
+    public OperateToCommand(int x, int y, Job2dDriver driver) {
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).